Dockerfile.test-infra

```diff
@@ -22,5 +22,8 @@ RUN pip3 install --no-cache-dir -r /tmp/requirements.txt && \
 # TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
 RUN chmod uga+w /etc/passwd
 
+ARG OPENSHIFT_RELEASE_IMAGE=https://mirror.openshift.com/pub/openshift-v4/clients/oc/{OPENSHIFT_VERSION:-4.5}/linux/oc.tar.gz
```

Comment:
Unfortunately I don't it is supported as well on Prow :(
Please transfer this oc extraction part to `scripts/install_environment.sh` script. So the argument will be passed on container runtime and not on container build time

Reply:
not sure he can move it to scripts/install_environment.sh cause he needs it as part of image. 
but maybe we can mount it?
@RazRegev ?

Reply:
@YuviGold can u validate we can live with this change in Prow or not?

Reply:
I will change that but I don't understand what is the problem with it